COPYING

```diff
@@ -0,0 +1,234 @@
+
+ We would like to clarify that we are using a convention where multiple names in
```

Comment:
Thanks for all this!!
Can we use the MIT license instead?
where it says:
Copyright (c) [year]  [fullname]
just put
Copyright (c) 2020- The Authors (see individual files for names)

Reply:
Of course we can. I will change it today.

PS: I am able to contribute only **out of working hours**, so I may be slow in response.

Reply:
Done.